bench/oracle-gas-model/build-model-page.py

```
@@ -0,0 +1,105 @@
+    The name may itself contain commas, so the six numeric fields are taken from the end.
+    """
+    out = {}
+    for line in path.read_text().splitlines():
+        line = line.strip()
+        if "ENG3913ROW," not in line:
+            continue
+        parts = line[line.index("ENG3913ROW,") + len("ENG3913ROW,"):].split(",")
+        nums = [int(p) for p in parts[-6:]]
+        name = ",".join(parts[:-6]).strip()
+        out[name] = dict(zip(
+            ["callGas", "overheadGas", "execGas", "calldataBytes", "calldataGas", "txTotal"], nums))
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)